containers/ddev-dbserver/Dockerfile

```diff
@@ -21,14 +21,15 @@ ARG MYSQL_ROOT_PASSWORD=root
 
 SHELL ["/bin/bash", "-c"]
 
-# Debian Stretch archives have been turned off
-RUN if grep "Debian GNU/Linux 9" /etc/issue >/dev/null ; then \
-    echo "deb http://archive.debian.org/debian/ stretch main contrib non-free" >/etc/apt/sources.list; \
+# Fix APT for Debian Stretch (EOL; upstream mirrors disabled)
+# Based on: https://serverfault.com/a/1131653
+RUN if grep "Debian GNU/Linux 9" /etc/issue >/dev/null; then \
```

Comment:
This might be better/easier with a heredoc these days, no need to change unless you want to. https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/